controllers/utils/index.js

```diff
@@ -37,7 +39,38 @@ const result = {
   }
 }
 
+function indexOfObject (array, field, value) {
+  for (let idx in array) {
+    if (array[idx][field] === value) {
+      return idx
+    }
+  }
+  return -1
+}
+
+function getUserIdByToken (tkn) {
```

Comment:
Esta función no debería estar aquí, ya existe getById en el código del token. Reemplazar cualquier uso de esta función por la nombrada (usar 'find usages' en webstorm)

Reply:
this es también parte de lo que decía yo, de refactorizar esto